deltalayer/sa.tf

```
@@ -0,0 +1,19 @@
+# Service account for the "streamer" cloud function
+resource "google_service_account" "sa_streamer" {
+  count = var.enable && contains(["preview_shared"], var.env_type) ? 1 : 0
```

```suggestion
  count = var.enable && contains(["preview_shared"], var.env_type) ? 1 : 0
```
```suggestion
  count = var.enable ? 1 : 0
```
the `contains(["preview_shared"]...` part of this conditional isn't needed here. That is a remnant from an older tf pattern we are moving away from